src/Frame.cpp

```diff
@@ -79,7 +79,9 @@ void Frame::guess_bonds() {
             }
             auto d = distance(i, j);
             auto radii = i_radius.value() + j_radius.value();
-            if (0.03 < d && d < 0.6 * radii && d < cutoff) {
+            // This criteria comes from VMD, updated with the -0.4 since we
+            // don't have the same VdW radii (see issue #301)
+            if (0.03 < d && d < 0.6 * (radii - 0.4) && d < cutoff) {
```

Comment:
There's something off here. Some builds pass all tests, others fail. Maybe the `0.4` magic number needs to be tweaked? For the ones that fail, a bond is missing/no bonds are found.

Reply:
The builder that pass the tests run with `CHFL_BUILD_DOCTESTS=OFF` since they are already very slow. The failing tests are doctests/examples that only run on the remaining ones. 

Since the doc tests use artificial positions, I would rather change them and keep the 0.4 here. See https://github.com/chemfiles/chemfiles/issues/301#issuecomment-574100048 for the rational.

Reply:
Update: looks like the Cl-Cl bond (which is the one being tested here) is actually 1.98 A. So this fix will not be enough here. 

I'll check if we can use an algorithm based on covalent radii instead of VdW radii for bond detection, since it would make more sense.